website/docs/patterns/multi-select-patterns/partials/interaction-concepts/interaction-concepts.md

```diff
@@ -0,0 +1,52 @@
+When combining a Multi-select pattern with filtering, sorting, pagination, etc, you _will_ encounter confusing scenarios about how selected results should be persisted across different pages and what the user expects when traversing large data sets.
```

Comment:
+1 

Also what about instead of "will" -> "may" and instead of "confusing" -> "complex"? The current wording feels a little negative and doesn't assume we know what the consumer finds "confusing".